### PR TITLE
Slack winston transport update

### PR DESCRIPTION
**Pull request by chrismaree**
This PR builds towards improving how we handle slack messages to be general based on the winston logging context. specifically this PR encodes
1) any base key `tx` key as a transaction from a winston. log for example the following tx key value pair in the logger object will be encoded as a URL.
```
    Logger.debug({
      at: "example",
      message: "this shows how to make a tx url",
      tx: `0xac8bef1f9556904042195c0372b22cf4ec37e5ca11e602e88814e45a8bd7d969`
    });
```
2) any `markwn` keyed field will have no formatting added, passing the value directly to the winston slack log. For example:
```
    Logger.debug({
      at: "example",
      markwn: "*this shows how to pass* markdown _to a message_",
    });
```

### CODE REVIEW

File: financial-templates-lib/logger/SlackTransport.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const slackFormatter = info => {
         type: "section",
         text: {
           type: "mrkdwn",
-          text: `*${info.level}: ${info.at}* --> ${info.message}`
+          text: `*${info.level}: ${info.at}* ⭢ ${info.message}`
```
Comment (nicholaspai) [inline financial-templates-lib/logger/SlackTransport.js:31]: smol arrow !

File: financial-templates-lib/logger/SlackTransport.js [hunk 2]
```diff
@@ -67,14 +67,33 @@ const slackFormatter = info => {
...
+      // if the key is a transaction object, generate a transaction link
+      if (key == "tx") {
+        formattedResponse.blocks.push({
+          type: "section",
+          text: {
+            type: "mrkdwn",
+            text: ` • _tx_: <https://etherscan.io/tx/${info[key]}|${info[key]}> \n`
+          }
+        });
+      } else if (key == "markwn") {
```
Comment (nicholaspai) [inline financial-templates-lib/logger/SlackTransport.js:79]: nit: Why `markwn` instead of `markdwn`?

Comment (chrismaree) [reply financial-templates-lib/logger/SlackTransport.js]: `markwn` is the key used in the Slack transport to define the `type`. I'm just following their convention.

**Comment by chrismaree** [comment]
> Looks good, one small nit on choice of key name.
> 
> Follow up question: is it possible to add simple unit tests to our Logger module just to test connectivity to different transports? or specifically for the Slack transport that certain key types (`tx`, `markdwn`) are handled correctly?

Ye, I think this is a good idea. will add this in a future review. Have been thinking about this for testing the monitor bot as well.

File: financial-templates-lib/logger/SlackTransport.js [hunk 3]
```diff
@@ -67,14 +67,33 @@ const slackFormatter = info => {
...
-          text: ` • _${key}_: ${info[key]}`
-        }
-      });
+      // if the key is a transaction object, generate a transaction link
+      if (key == "tx") {
+        formattedResponse.blocks.push({
+          type: "section",
+          text: {
+            type: "mrkdwn",
+            text: ` • _tx_: <https://etherscan.io/tx/${info[key]}|${info[key]}> \n`
```
Comment (mrice32) [inline financial-templates-lib/logger/SlackTransport.js:76]: +1

File: financial-templates-lib/logger/SlackTransport.js [hunk 4]
```diff
@@ -67,14 +67,33 @@ const slackFormatter = info => {
...
+      } else if (key == "mrkdwn") {
+        formattedResponse.blocks.push({
+          type: "section",
+          text: {
+            type: "mrkdwn",
+            text: ` ${info[key]}`
+          }
+        });
+      } else {
+        // If the value in the message object is an string or a integer then show it as _key: value
```
Comment (mrice32) [inline financial-templates-lib/logger/SlackTransport.js:88]: what do you mean by the underscore in `_key: value`?

**Pull request merged**